### ISSUE TITLE
Release & Publish latest changes to RubyGems

### ISSUE THREAD

**Issue by NickMeves** [SumoLogic/logstash-output-sumologic#42]
Been a while since a release for getting the new features easily with `bin/logstash-plugin install logstash-output-sumologic`

Lots of improvements merged into master over the last few months.

The `event.sprintf` headers improvements particularly interests me and I would love to easily just use logstash-plugin install.

Thanks!

**Comment by vmiszczak-teads** [comment]
I've been trying `logstash-plugin install --version=1.3.0 logstash-output-sumologic` and it's broken. I need dynamic host name from this patch. 
```
Plugin version conflict, aborting
ERROR: Installation Aborted, message: Bundler could not find compatible versions for gem "logstash-mixin-http_client":
  In snapshot (Gemfile.lock):
    logstash-mixin-http_client (= 7.0.0)

  In Gemfile:
    logstash-input-http_poller java was resolved to 5.0.1, which depends on
      logstash-mixin-http_client (~> 7) java

    logstash-output-sumologic (= 1.3.0) java was resolved to 1.3.0, which depends on
      logstash-mixin-http_client (~> 6.0) java
```

**Comment by astencel-sumo** [comment]
Hey folks, sorry for the long wait. This took way too long, but here it is at last: just released version [v1.3.1](https://github.com/SumoLogic/logstash-output-sumologic/releases/tag/v1.3.1) which should hopefully fix the above error and work with Logstash 7 without issues. I'll close this issue now, please open a new issue if there are any problems.
